substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/Target_jdk_jfr_internal_JVM.java

```diff
@@ -198,6 +199,12 @@ public static long getTicksFrequency() {
         return JfrTicks.getTicksFrequency();
     }
 
+    @Substitute
+    @TargetElement(onlyWith = JDK24OrLater.class)
+    public static long nanosNow() {
+        return JfrTicks.currentTimeNanos();
```

Comment:
Is the accuracy of this high enough? This method only offers milliseconds resolution.